package.json

```diff
@@ -12,7 +12,7 @@
     "node": ">=8.10.0"
   },
   "dependencies": {
-    "chosen-js": "^1.8.7",
+    "accessible-autocomplete": "^2.0.3",
```

Comment:
Looks like we’re loading this from `vendor/` rather than `node_modules` — is this needed here?

Reply:
@hbillings thoughts?